.nvmrc

```diff
@@ -0,0 +1 @@
+v7.10.1
```

Comment:
Are we now recommending at least Node 7.10?

Reply:
nah, this was to get netlify to use a version that would work with nodegit; it didn't actually help (i removed nodegit), so i might be able to remove this file. For the moment, saying we support active LTS versions of node should be adequate, though we're not currently setup to test against that many node versions in a reasonable amount of time.